zebra/zebra_nb.c

```diff
@@ -358,6 +358,13 @@ const struct frr_yang_module_info frr_zebra_info = {
 				.destroy = lib_interface_zebra_mpls_destroy,
 			}
 		},
+		{
+			.xpath = "/frr-interface:lib/interface/frr-zebra:zebra/link-params",
+			.cbs = {
+				.create = lib_interface_zebra_link_params_create,
+				.destroy = lib_interface_zebra_link_params_destroy,
```

Comment:
Having just read the NB conversion doc it specifically mentioned using presence contains like this. One thing it also mentioned that's missing here is possible use of `.apply_finish` callback. This gets called after all the applies have completed for children. This would be particularly useful here b/c then only a single call to `zebra_interface_parameters_update` can be done (so only one message is sent out even with multiple changes). I think we should consider this in a follow on PR

Reply:
Yes it totally makes sense, I will do it in a separate PR or in an additional commit to this one if there are any problems found during the review.